### ISSUE TITLE
Constant regexp matching

### ISSUE THREAD

**Issue by mxstbr** [withspectrum/slate-mentions#1]
Should use constant RegExp matching (like [`slate-auto-replace`](https://github.com/ianstormtaylor/slate-auto-replace)) for matching `@`. (or any other character)

**Comment by oyeanuj** [comment]
As part of this, could also consider having the trigger be an option (with the default being '@'). I am thinking of cases where one might want to use it for hashtag and it would be a simple drop-in replacement.

**Comment by mxstbr** [comment]
Yep, for sure. This would be a regexp that you can supply, so it could be a hashtag, an `@`, whatever. We have that usecase, so that'll definitely be an option!
